src/gulpfile.js

```diff
@@ -4,6 +4,8 @@ const debug = require('debug')('slate-tools');
 const argv = require('yargs').argv;
 const runSequence = require('run-sequence');
 
```

Comment:
@chrisberthe interesting haha. Same as above then... If there's already a pattern established in the codebase, it is probably better to stick to that.